Owncloud iOs Client/Cells/CustomCell/CustomCellFileAndDirectory.h

```diff
@@ -25,7 +25,7 @@
 @property(nonatomic, weak) IBOutlet UIImageView *sharedByLinkImage;
 @property(nonatomic, weak) IBOutlet UIImageView *sharedWithUsImage;
 
-@property(nonatomic, weak) NSURLSessionTask *thumbnailSessionTask;
+@property(nonatomic, strong) NSURLSessionTask *thumbnailSessionTask;
```

Comment:
Yes, it was, and also I am going to update the previous IBOutlet properties to strong too, maybe in the apple documentation is not updated but as the [WWDC2015](https://developer.apple.com/videos/play/wwdc2015/407/) they suggest to use strong references for performance due the deallocated process, and since Xcode7 create strong as default.